src/dataExplorer/components/SaveAsCellForm.tsx

```diff
@@ -170,21 +176,42 @@ class SaveAsCellForm extends PureComponent<Props, State> {
 
     const viewWithProps: View = {...view, name: cellName}
 
+    let succeeded = false
+    const redirectHandler = (dashboardId: string): void =>
+      history.push(`/orgs/${orgID}/dashboards/${dashboardId}`)
+
     try {
-      targetDashboardIDs.forEach(dashID => {
+      targetDashboardIDs.forEach((dashID, idx) => {
```

Comment:
```suggestion
      targetDashboardIDs.forEach((dashID, i) => {
```

one of the few times it's better to just use a single letter since it's conventional across the industry. everybody knows what `i` represents. and it usually clues you in that you're in a loop. again, special case because it's conventional across many languages and paradigms.